packages/replay-internal/src/util/addEvent.ts

```diff
@@ -83,6 +83,8 @@ async function _addEvent(
     DEBUG_BUILD && logger.error(error);
```

Comment:
let's replace this line instead with the `handleException` call, I'd say? Otherwise this would double-log as of now (and it probably also makes sense to log/capture this before the await, I think?)